.github/workflows/fvt.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.18.7'
+          go-version: '1.19'
```

Comment:
can we stick with `go-version` `1.18.7` here? Upgrading to 1.19 requires many more changes to go source files. This is likely what causes `lint` checks to fail currently